sdks/python/apache_beam/io/gcp/tests/pubsub_matcher.py

```diff
@@ -38,7 +38,6 @@
   pubsub = None
 
 DEFAULT_TIMEOUT = 5 * 60
-MAX_MESSAGES_IN_ONE_PULL = 50
```

Comment:
We can rename this constant to `DEFAULT_MAX_MESSAGES_IN_ONE_PULL` and use as a default parameter value.